packages/expo/e2e/__tests__/utils.ts

```diff
@@ -110,7 +112,7 @@ export async function createFromFixtureAsync(
     console.log('[setup] Created fixture project:', projectRoot);
 
     // Copy all files recursively into the temporary directory
-    await fs.copySync(fixturePath, projectRoot);
+    await copySync(fixturePath, projectRoot);
```

Comment:
In Node 16.7 there's fs.cp which recursively copies but can't assume it will be present on the user's computer for awhile